qiskit/pulse/schedule.py

```diff
@@ -397,13 +398,11 @@ def __lshift__(self, time: int) -> 'Schedule':
         return self.shift(time)
 
     def __repr__(self):
-        res = 'Schedule("name=%s", ' % self._name if self._name else 'Schedule('
-        res += '%d, ' % self.start_time
-        instructions = [repr(instr) for instr in self.instructions]
-        res += ', '.join([str(i) for i in instructions[:50]])
-        if len(instructions) > 50:
-            return res + ', ...)'
-        return res + ')'
+        name = "name={}, ".format(self._name) if self._name else ""
```

Comment:
```suggestion
        name = "name={}, ".format(self._name) if self._name else "''"
```
This way we'd see `Schedule(name='', ...)` rather than `Schedule(name=, ...)`

Reply:
we will see `Schedule(...)` with no `name` as it was before :)